assets/app/views/_templateopt.html

```diff
@@ -22,12 +22,16 @@
           type="text"
           placeholder="{{ parameter | parameterPlaceholder }}"
           ng-model="parameter.value"
-          ng-required="parameter.required && !parameter.generate">
+          ng-required="parameter.required && !parameter.generate"
+          ng-trim="false">
     </span>
     <div class="help-block" ng-if="parameter.description">{{parameter.description}}</div>
     <div ng-show="paramForm[paramID].$error.required && paramForm[paramID].$touched" class="has-error">
       <div class="help-block">{{parameter.name}} is required.</div>
     </div>
+    <div ng-show="isOnlyWhitespace(parameter.value)" class="has-warning">
+      <span class="help-block">This parameter value only has spaces.</span>
```

Comment:
Realized it might not be clear what I meant, HTML-wise this is fine, I think the message needs to change but I don't have a good suggestion for what it should be